bulk-ingester/src/main/java/org/alfresco/hxi_connector/bulk_ingester/processor/model/Node.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * #%L
+ * Alfresco HX Insight Connector
+ * %%
+ * Copyright (C) 2024 Alfresco Software Limited
+ * %%
+ * This file is part of the Alfresco software.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail.  Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ * #L%
+ */
+
+package org.alfresco.hxi_connector.bulk_ingester.processor.model;
+
+import java.io.Serializable;
+import java.time.ZonedDateTime;
+import java.util.Map;
+import java.util.Set;
+
+import org.alfresco.elasticsearch.db.connector.model.ContentMetadata;
+
+public record Node(
```

Comment:
Could we instead map the DB data to an `IngestMetadataCommand` and an `IngestContentCommand` since these are already part of our internal model? Do you think that would be too much for one step?

Reply:
I dont think it makes any sense as we have to send the events anyway so it will be easier to have some custom class we can easily serialise/deserialise

Reply:
I was thinking we could serialise/deserialise those command classes. I'm happy with this approach too, but eventually we know we want to get those two classes so I thought it might be simpler overall.

Reply:
there is this `PropertyDelta` wrapper around all properties in `IngestMetadataCommand` so imo it will be hard to write some serialization logic that makes sense :/ Also we would have to somehow import `live-ingester` to `bulk-ingester` which may cause with some coupling we would not like to have So I think leaving this class is just much simpler approach

Reply:
we can always refactor it later if needed